src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -132,8 +112,9 @@ private void addNewNode(K key, V value, int arrayIndex) {
         } while (true);
     }
 
-    private int thresholdCalculation(int ArraySize) {
-        return threshold = (int) (ArraySize * LOAD_FACTOR);
+    private int calculateThreshold(int size) {
+        threshold = (int) (size * LOAD_FACTOR);
+        return threshold;
     }
```

Comment:
your method named "calculateThreshold", so you need here just to calculate it and return, don't change

Reply:
@al-pavlenko +